Fixing the Supermarket API documentation.

iennae opened the pull request:
There is a subtle difference between the sets of controllers available on the Supermarket
between https://supermarket.chef.io/cookbooks and https://supermarket.chef.io/api/v1.

https://supermarket.chef.io/cookbooks, /search, /tools are the html interface that users interact directly with via the website. The API that folks can curl against are what is behind /api/v1/INTERFACENAME.

If someone were to use the /cookbooks interface versus /api/v1/cookbooks for example they will have a different experience. One example is that /cookbooks doesn't have a user query term but /api/v1/cookbooks does.

Reference: [https://github.com/chef/supermarket/tree/master/src/supermarket/app/controllers](https://github.com/chef/supermarket/tree/master/src/supermarket/app/controllers)

Before merging, @robbkidd should give a 👍 .

/universe is the one intentional API endpoint not modified as it is available on /universe and returns a JSON response as expected.

chef-delivery commented:
This PR has passed 'Verify' and is ready for review and approval!
  Use: '@delivery approve' when code review is complete.

jamescott commented:
@delivery approve

chef-delivery commented:
Failed to approve change: dae3e8eb-7a88-4f73-8471-fb894222ea0b
  Change is already merged. @jamescott

chef-delivery commented:
This PR has passed 'Acceptance' and is ready to be delivered!
  Use: '@delivery deliver' when validated in acceptance.

iennae committed:
Fixing the Supermarket API documentation.

There is a subtle difference between the sets of controllers available on the Supermarket
between https://supermarket.chef.io/cookbooks and https://supermarket.chef.io/api/v1.

https://supermarket.chef.io/cookbooks, /search, /tools are the html interface that users interact directly
with on the website. The API that folks can curl against are what is behind /api/v1/INTERFACENAME.

If someone were to use the /cookbooks interface versus /api/v1/cookbooks for example they will have a
different experience. This is seen by the fact that /cookbooks doesn't have a user query term.

Reference: https://github.com/chef/supermarket/tree/master/src/supermarket/app/controllers

Signed-off-by: Jennifer Davis <iennae@gmail.com>

robbkidd commented on chef_master/source/supermarket_api.rst:
```diff
@@ -807,5 +806,10 @@ The response will return details for a user, including their name, Chef username
+     }, 
+     "tools": {
+       "owns": {
+         "bacon_tool": "https://supermarket.chef.io/api/v1/tools/bacon_tool"
```
Mmmm. 🥓

robbkidd reviewed (approved):
I laughed. I cried. It was the "feel good" PR of the summer.

:shipit:

brewn commented:
![tenor-124157559](https://user-images.githubusercontent.com/21040805/27198421-17e472da-51e0-11e7-90a8-c3e5b1572548.gif)
